hat/app/org/hatdex/hat/utils/LoggingFilter.scala

```diff
@@ -47,4 +52,39 @@ class LoggingFilter @Inject() (
     configuration: Configuration,
     implicit override val mat: Materializer,
     implicit override val ec: ExecutionContext)
-    extends Slf4jLoggingFilter(configuration, Component.Hat)
+    extends Slf4jLoggingFilter(configuration, Component.Hat) {
+
+  private val authTokenFieldName: String =
+    configuration.get[String]("silhouette.authenticator.fieldName")
+
+  override val body: Some[RequestHeader => JsValue] = Some((requestHeader: RequestHeader) =>
+    Json.parse(
+      requestHeader.queryString
+        .get(authTokenFieldName)
+        .flatMap(_.headOption)
+        .orElse(requestHeader.headers.get(authTokenFieldName))
+        .flatMap(t =>
+          if (t.isEmpty) None
+          else Some(t)
+        )
+        .flatMap(t => Try(JWSObject.parse(t)).toOption)
+        .map(o => JWTClaimsSet.parse(o.getPayload.toJSONObject))
+        .map { claimSet =>
+        s"""
+        {
```

Comment:
The node `request` -> `pda` is missing.
It should contain the fully qualified HAT url.

Reply:
apologies i thought we decided that wasnt needed will add immediately

Reply:
no worries.

LGTM here.